Assets/Scripts/Screeps3D/NukeMonitor.cs

```diff
@@ -206,34 +204,34 @@ private IEnumerator GetNukes()
                         });
 
 
-                    /* Example
```

Comment:
Euhmm.. this was already commented out with a multiline comment xD e.g. /* ... */

Reply:
must have left after my big comment-out-everything investigation